src/Api/Controller/ListUsersController.php

```diff
@@ -76,6 +76,15 @@ protected function data(ServerRequestInterface $request, Document $document)
 
         $actor->assertCan('viewUserList');
 
+        if (! $actor->hasPermission('user.viewLastSeenAt')) {
+            // If a user cannot see everyone's last online date, we prevent them from sorting by it
+            // Otherwise this sort field would defeat the privacy setting discloseOnline
+            // We use remove instead of add so that extensions can still completely disable the sort using the extender
+            $this->removeSortField('lastSeenAt');
+        }
+
+        $query = Arr::get($this->extractFilter($request), 'q');
```

Comment:
You might have messed up the merge. I think this line has disappeared from master